apps/condo/domains/notification/templates.js

```diff
@@ -1,278 +1,176 @@
-                html: `
-                    <html style="fmargin: 0; font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif; font-size: 22px; font-weight: 400; line-height: 32px; text-align: left;">
-                        <body style="margin: 0; font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif; font-size: 22px; font-weight: 400; line-height: 32px; text-align: left;">
-                            <table width="100%" border="0" cellpadding="0" cellspacing="0" style="margin:0; padding:0">
-                                <tr>
-                                    <td width="10%" align="left"><b style="font-size: 16px;">DOMA.ai</b></td>
-                                    <td width="80%">&nbsp;</td>
-                                    <td width="10%" align="right"><b style="font-size: 16px;">ДИСПЕТЧЕРСКАЯ</b></td>
-                                </tr>
-                            </table>
-                            <p style="font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif; font-size: 22px; font-weight: 400; line-height: 32px; text-align: left;">Добрый день!<br />
-                            С вами поделились заявкой №${ticketNumber} от ${dayjs(date).locale(LOCALES[RU_LOCALE]).format('D MMMM YYYY')}.<br />
-                            Текст заявки: «${details}»</p>
-                            <p>&nbsp;</p>
-                            <div><!--[if mso]>
-                            <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" href="${serverUrl}/ticket/${id}" style="height:40px;v-text-anchor:middle;width:330px;" arcsize="10%" stroke="f" fill="t">
-                                <v:fill type="tile" src="" color="#4CD174" />
-                                <w:anchorlock/>
-                                <center style="color:#ffffff;font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif;font-size:16px;font-weight:bold;">Перейти в карточку заявки в Doma.ai</center>
-                            </v:roundrect>
-                            <![endif]--><a href="${serverUrl}/ticket/${id}" style="background-color:#4CD174;border-radius:4px;color:#ffffff;display:inline-block;font-family: Roboto, Arial, 'Nimbus Sans L', Helvetica, sans-serif;font-size:16px;font-weight:bold;line-height:40px;text-align:center;text-decoration:none;width:330px;-webkit-text-size-adjust:none;mso-hide:all;">Перейти в карточку заявки в Doma.ai</a></div>
-                        </body>
-                    </html>
-                `,
-            }
+/**
+ * Template environment variable type
+ * @typedef {Object} MessageTemplateEnvironment
+ * @property {string} serverUrl - server url may use for building links
+ */
+
+/**
+ *
+ * @type {Object<string, function({message: Message, env: MessageTemplateEnvironment}): Object>}
+ */
+const MESSAGE_TRANSPORTS_RENDERERS = {
+    [SMS_TRANSPORT]: function ({ message, env }) {
```

Comment:
inlined functions are harder to read and understand, I'd move them out from object.

Reply:
Ok, done.